capistrano-rails.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
 
   gem.add_dependency 'capistrano', '~> 3.1'
-  gem.add_dependency 'capistrano-bundler', '~> 1.1'
+  gem.add_dependency 'capistrano-bundler', '>= 1.1'
```

Comment:
Would it make sense to do something like `'>= 1.1', '< 3'`? That way if the next breaking change also requires a newer version of `capistrano-rails`, we'll be set.

Reply:
Makes sense!